libs/http-ext/src/lib/matchers/match-origin/match-origin.spec.ts

```diff
@@ -4,21 +4,37 @@ import { matchOrigin } from './match-origin';
 describe.each([
   /* Using a string */
   ['https://test.com', 'https://test.com', true],
+  // @todo ['https://test.com/test', 'https://test.com', true],
```

Comment:
We should enable these tests to make sure that we check agains the origin and not the URL.